inc/vimcat_functions.sh

```diff
@@ -19,7 +19,7 @@ quit() {
 
         rm -rf "$tmp" 2>/dev/null # rm -rf "" shows error on OpenBSD
     ) &
-    exit "$@"
+    exit "${@:-0}"
```

Comment:
I will also change this to `exit "${1:-0}"` as e.g. `exit 1 2` fails. Some interactive shells don't even exit if you give more than one argument to `exit`.